Makefile

```diff
@@ -16,7 +16,7 @@ GIT_COMMIT:=$(shell git rev-parse --short HEAD)
 REPO_DIR:=$(shell pwd)
 
 # for testing, the built image will also be tagged with this name
-OVERRIDE_IMAGE_NAME?=vikramraman/wavefront-collector:0.9.9-rc3
+OVERRIDE_IMAGE_NAME?=glaullon/wavefront-collector:latest
```

Comment:
Modified master to use env vars for this: https://github.com/wavefrontHQ/wavefront-kubernetes-collector/commit/6e993435ae5cc7be37c36b5edb58c19d02d70696

Reply:
Merge master and revert this line.